db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+users = User.create!([{email: "mail4@test.pl", first_name: "Kermit", last_name: "Frog",
```

Comment:
In that case you don't need to asign operation result to any variable.

Reply:
Ok, I through that we have to assign it to variable if we want have to acces to this content from another place in code. If not, I have just removed it.